mimic/test/test_core.py

```diff
@@ -55,3 +59,6 @@ def test_from_plugin_includes_all_plugins(self):
             len(plugin_apis),
             len(list(core.entries_for_tenant('any_tenant', {},
                                              'http://mimic'))))
+        self.assertEqual(
+            len(domain_apis),
+            len(list(core.domains)))
```

Comment:
@ksheedlo - I've created (but not yet pushed or finished implementing) a test that writes a plugin implementing IAPIDomainMock, loads it, and messes with the path to make it visible. I'm most interested in testing that plugins implementing the interface are indeed loaded by `fromPlugins`.